microsoft-365/security/defender-endpoint/configure-siem.md

```diff
@@ -69,6 +69,15 @@ Using the Microsoft 365 Defender Add-on for Splunk that supports:
 
 For more information on the Microsoft 365 Defender Add-on for Splunk, see [splunkbase](https://splunkbase.splunk.com/app/4959/).
 
+### Datadog
+
+Microsoft 365 Defender for Endpoint Integration with Datadog supports:
+
+- Ingesting Microsoft Defender for Endpoint alerts and Incidents
+- Dashboards that enable monitoring metrics across Endpoint, Threats & Vulnerabilities, and Software
```

Comment:
```suggestion
- Ingesting Microsoft Defender for Endpoint alerts and incidents
- Dashboards that enable monitoring metrics across endpoint, threats and vulnerabilities, and software
```

Reply:
Updated in commit id [100e518](https://github.com/MicrosoftDocs/microsoft-365-docs/pull/9189/commits/100e51816e641f96ca1ccf49d756bbd7bae76adb)